indico/modules/events/tracks/templates/management.html

```diff
@@ -38,16 +38,15 @@
         </div>
     </div>
 
-    {# TODO: Use new flex classes instead of the inline CSS once they are available #}
-    <div class="flex" style="flex-direction: row;">
-        <h3 style="flex-grow: 1; margin-bottom: 0;">{% trans %}List of tracks{% endtrans %}</h3>
-        <div style="margin: 1em 0 0 0;">
+    <div class="flexrow f-a-center">
+        <h3 class="stretch">{% trans %}List of tracks{% endtrans %}</h3>
+        <div>
```

Comment:
No, but it should normally be a toolbar... However, in this case since the button is only one, we could work with just that (without a toolbar)